plugin/package.json

```diff
@@ -43,11 +43,10 @@
   "dependencies": {
     "ace-builds": "^1.15.0",
     "jquery": "^3.6.1",
-    "jquery-ui": "^1.13.2",
-    "raf": "^3.4.1"
```

Comment:
This was only needed before to provide `window.requestAnimationFrame` support to browsers we no longer support: https://caniuse.com/requestanimationframe